src/stklos.c

```diff
@@ -103,6 +103,7 @@ static void Usage(char *progname, int only_version)
 "       --case-insensitive      be case incensitive by default\n"
 "   -u, --utf8-encoding=yes|no  use/don't use UTF-8 encoding (instead of default)\n"
 "   -v, --version               print program version and exit\n"
+"   -V, (SRFI-176)              print version and program information, as per SRFI-176"
```

Comment:
Missing `\n` and says `SRFI-176` twice.

```
"   -V                          print version and program information, as per SRFI-176\n"
```

Reply:
I have corrected this and several other things:

- To be more coherent with other programs stklos -v doesn' t print the path
  name anymore, and the version number is not in parentheses
- the `--version` indicates now that you can find more information with -V
- `version-alist` has been moved in `bonus.stk` since the support for other  inlined SRFI is here
- added (minimal) documentation for `version-alist`
- Usage information was printed on stderr. It is printed now on stdout as usual
- Only incorrect options are displayed on stderr
- the test of the -V option arrives too late (test was done after tring to create STklos configuration directory)

Reply:
Since `-V` is different from the other version options, and `-V` didn't exist before this patch, should we remove the redundant first line `stklos 1.40` from the `-V` output? It has `(version "1.40")` and `(command "stklos")` which gives the same information in a standard format.